### PR TITLE
Add new Spring and Junit resources

### PR DESCRIPTION
**Pull request by baktiyarov21**
* Add new Spring resources from sysout.ru
* Add new Junit resource from coderlessons.com

## For resources
Sysout.ru - Java, Spring and Git tutorials
### Description
Learn functionality of Spring Framework componets

### Why is this valuable (or not)?
Novice Java developers can learn to work with the framework

### How do we know it's really free?
Anyone can go to their website and use the examples to learn. Each theme has its own code on github

### For book lists, is it a book? For course lists, is it a course? etc.
It's site with tutorials.

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/master/CONTRIBUTING.md)
- [x] Search for duplicates.
- [x] Include author(s) and platform where appropriate.
- [ ] Put lists in alphabetical order, correct spacing.
- [ ] Add needed indications (PDF, access notes, under construction)

### CODE REVIEW

File: books/free-programming-books-ru.md [hunk 1]
```diff
@@ -271,6 +271,7 @@
 
 * [Руководство по JUnit](http://proselyte.net/tutorials/junit) - Евгений Сулейманов
 * [JUnit 4](https://easyjava.ru/testirovanie/junit-2/)
+* [JUnit Учебник](https://coderlessons.com/tutorials/java-tekhnologii/vyuchit-junit/junit-uchebnik) - CoderLessons.com
```
Comment (LuigiImVector) [inline books/free-programming-books-ru.md:274]:
```suggestion
* [JUnit Учебник](https://coderlessons.com/tutorials/java-tekhnologii/vyuchit-junit/junit-uchebnik) - CoderLessons
```

File: books/free-programming-books-ru.md [hunk 2]
```diff
@@ -285,7 +286,9 @@
 * [Руководство по Spring](http://proselyte.net/tutorials/spring-tutorial-full-version) - Евгений Сулейманов
 * [Spring Framework](https://easyjava.ru/spring/)
 * [Spring MVC Tutorial](https://coderlessons.com/tutorials/java-tekhnologii/uchis-spring-mvc/spring-mvc-tutorial) - CoderLessons.com
-
+* [Spring Core](https://sysout.ru/spring/spring-core/) - Sysout.ru
+* [Spring Data](https://sysout.ru/spring/spring-data/) - Sysout.ru
+* [Spring Security](https://sysout.ru/spring/spring-security/) - Sysout.ru
```
Comment (LuigiImVector) [inline books/free-programming-books-ru.md:291]:
```suggestion
* [Spring](https://sysout.ru/spring/) - Sysout
* [Spring Framework](https://easyjava.ru/spring/) - EasyJava
* [Spring MVC Tutorial](https://coderlessons.com/tutorials/java-tekhnologii/uchis-spring-mvc/spring-mvc-tutorial) - CoderLessons

```

**Review by LuigiImVector — COMMENTED** [review]
I think that is better add only the main link

**Comment by LuigiImVector** [comment]
This resource has too many ads, should it be removed?
https://github.com/EbookFoundation/free-programming-books/blob/52cbb513ed81a8aba8e949f975d653ca62ba2466/books/free-programming-books-ru.mdL274

**Comment by eshellman** [comment]
looks like a blog to me

**Comment by eshellman** [comment]
Closed because resource out of scope

**Pull request closed without merging**